From e5132304dc46089b8709000a222a42aea7731f32 Mon Sep 17 00:00:00 2001
From: Ondrej Jirman <megous@megous.com>
Date: Tue, 23 Jun 2020 19:17:37 +0200
Subject: [PATCH 024/312] ARM: dts: sun8i-a83t-tbs-a711: Don't increase BT baud
 rate to 1.5MHz

Since Linux 5.6, increasing the baud rate makes the bt_bcm fail to
switch frequencies for reasons outside of BT driver changes between
v5.5 and v5.6.

Reasons are unknown.

Signed-off-by: Ondrej Jirman <megous@megous.com>
---
 arch/arm/boot/dts/sun8i-a83t-tbs-a711.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/sun8i-a83t-tbs-a711.dts b/arch/arm/boot/dts/sun8i-a83t-tbs-a711.dts
index 491aa409fc46..71d5923d2e34 100644
--- a/arch/arm/boot/dts/sun8i-a83t-tbs-a711.dts
+++ b/arch/arm/boot/dts/sun8i-a83t-tbs-a711.dts
@@ -593,7 +593,7 @@ bluetooth {
 		device-wakeup-gpios = <&pio 7 5 GPIO_ACTIVE_HIGH>; /* PH5 */
 		host-wakeup-gpios = <&r_pio 0 5 GPIO_ACTIVE_HIGH>; /* PL5 */
 		shutdown-gpios = <&r_pio 0 4 GPIO_ACTIVE_HIGH>; /* PL4 */
-		max-speed = <1500000>;
+//		max-speed = <1500000>; stopped working in v5.6
 	};
 };
 
-- 
2.31.1

